src/main/kotlin/no/nav/helsearbeidsgiver/config/ApplicationConfig.kt

```diff
@@ -268,11 +264,11 @@ fun Application.configureAuth(authClient: AuthClient) {
     }
 }
 
-fun configurePdpService(authClient: AuthClient): IPdpService =
+fun getPdpService(): IPdpService =
```

Comment:
Jeg antar at object gjør at det ikke blir instansiert en ny service for hvert kall?

Reply:
Stemmer!